Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableToggleCollectionInspector.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Editor
+{
+    [CustomEditor(typeof(InteractableToggleCollection))]
+    public class InteractableToggleCollectionInspector : UnityEditor.Editor
```

Comment:
@julenka, what do you want to do with inspector public class comments? Mark internal or just add generic comment "I am inspector of blah"

Reply:
I like marking them as internal

Reply:
@Troy-Ferrell and @julenka How do I mark something as internal?

Reply:
`internal class InteractableToggleCollectionInspector `